docs/build-parachains.md

```diff
@@ -0,0 +1,174 @@
+---
+id: build-parachains
+title: Parachain Development
+sidebar_label: Parachain Development
+---
+
+This section will cover the motivation to build a parachian or parathread, the tools available
+to facilitate this, and the steps to test, and finally launch your network on Polkadot!
```

Comment:
Perhaps: "to facilitate this, the steps to test, and finally, how to launch your network on Polkadot"